session.go

```diff
@@ -631,6 +632,16 @@ func (s *session) handleTCPPackage() error {
 	pktBuf = gxbytes.NewBuffer(nil)
 
 	conn = s.Connection.(*gettyTCPConn)
+	if tlsConn, ok := conn.conn.(*tls.Conn); ok {
+		tlsHandshaketime := defaultTLSHandshakeTimeout
+		if s.ReadTimeout() > 0 {
```

Comment:
dubbo-go will set 1s by default, so it is recommended to use readTime first. If it is equal to 0, use the default 3s.